app/code/Magento/Checkout/Model/PaymentInformationManagement.php

```diff
@@ -85,12 +99,15 @@ public function savePaymentInformationAndPlaceOrder(
         try {
             $orderId = $this->cartManagement->placeOrder($cartId);
         } catch (\Magento\Framework\Exception\LocalizedException $e) {
+            /** @var \Magento\Quote\Model\Quote $quote */
+            $quote = $this->cartRepository->getActive($cartId);
+            $this->checkoutHelper->sendPaymentFailedEmail($quote, __($e->getMessage()));
```

Comment:
Could you please explain why we should send an email with payment failed status every time when placeOrder method returns exception? The exception can be caused by different reasons, not only if payment if failed.